app/jobs/autotitle_conversation_job.rb

```diff
@@ -1,11 +1,16 @@
+class ConverstionNotReady < StandardError; end
```

Comment:
I think I might try to move this down into the class rather than at the top level so it doesn't exist in the global namespace. Does that work?

Reply:
Ah, I thought it had to be top-level. But I just tried it within the class and it worked. Good suggestion